spaces/domains.json

```
@@ -78,6 +78,7 @@
   "gov.mettalex.com": "mettalex",
   "gov.moontools.io": "moontools",
   "voting.pancakeswap.finance": "pancake",
+  "vote.steakbank.finance":"steakbank.eth",
```

Please add a space here.
```suggestion
  "vote.steakbank.finance": "steakbank.eth",
```